### PR TITLE
Add product analyst levels

### PR DESCRIPTION
**Pull request by jasonblais**
Add product analyst levels.

Source: https://docs.google.com/spreadsheets/d/1D8cGVMvLv4qKSpev7R3_DqEeiry-uKCjVdwbCTTgA0I/edit?userstoinvite=richard.pidgeon%40mattermost.com&actionButton=1#gid=1878710208

For comparable PRs, see 
 - PM levels https://github.com/mattermost/mattermost-handbook/pull/884
 - Tech writing levels https://github.com/mattermost/mattermost-handbook/pull/885

### CODE REVIEW

**Review by wiersgallak — DISMISSED** [review]

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Product Analyst Levels
+
+These levels serve to help Product Analyst assess their opportunities for career growth in the Product Analytics function. The levels below follow a growth track in the art and craft of Product Analysts and not in a person management track.
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
These levels serve to help Product Analysts assess their opportunities for career growth in the Product Analytics function. The levels below follow a growth track in the art and craft of Product Analysts and not in a person management track.
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+The level descriptions below are transferrable to comparable analyst roles, such as a Product Analytics Engineer and a Data Analyst.
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
The level descriptions below are transferable to comparable analyst roles, such as a Product Analytics Engineer and a Data Analyst.
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.*| Defines new task assignments for members of other teams, usually without requiring help.<br><br>Mentors and trains new team members while leading the coordination and management of high impact initiatives. |
+| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Frequently sought out by product and engineering leads for opinion on data implementation decisions (both product and technical directions). |
+| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans across staff, open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Has ownership for cross-product customer impact topics. Can anticipate complex issues early in the planning and development processes. Starts to anticipate and resolve cross functional topics and issues. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans across staff, open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Has ownership for cross-product customer impact topics. Can anticipate complex issues early in the planning and development processes. Starts to anticipate and resolve cross-functional topics and issues. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 7: Staff Product Analyst
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Consults on architectural visions for data collection, modeling, and innovations across teams. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.*| Defines new task assignments for members of other teams, usually without requiring help.<br><br>Mentors and trains new team members while leading the coordination and management of high impact initiatives. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.*| Defines new task assignments for members of other teams, usually without requiring help.<br><br>Mentors and trains new team members while leading the coordination and management of high impact initiatives. |
```

Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.*| Defines new task assignments for members of other teams, usually without requiring help.<br><br>Mentors and trains new team members while leading the coordination and management of high impact initiatives. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 6: Senior Product Analyst
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Sets and delivers architectural vision for high impact features and changes across the data analytics infrastructure. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Defines new feature assignments for other members of the team, usually without requiring help.<br><br>Leads cross-functional groups, and coordinates and manages the creation of new functionality. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Defines new feature assignments for other members of the team, usually without requiring help.<br><br>Leads cross-functional groups, and coordinates and manages the creation of new functionality. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 5: Senior Product Analyst
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Sets and delivers architectural vision for high impact features and changes across the data analytics infrastructure. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Defines new feature assignments for themselves, usually without requiring help.<br><br>Organizes cross-functional initiatives in building significant new functionality.|
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Defines new feature assignments for themselves, usually without requiring help.<br><br>Organizes cross-functional initiatives in building significant new functionality.|
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 4: Product Analyst II
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.* | Designs, leads and delivers high impact features and changes across the data infrastructure stack. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives minimal instruction.<br><br>Participates in cross-functional groups towards building significant new functionality. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Receives minimal instruction.<br><br>Participates in cross-functional groups towards building significant new functionality. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 3: Product Analyst II
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.* | Designs, leads and delivers high impact features and changes across the data infrastructure stack. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives general instruction on new assignments or unfamiliar tasks.<br><br>Participates in cross-functional groups towards building significant new functionality. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Receives general instruction on new assignments or unfamiliar tasks.<br><br>Participates in cross-functional groups towards building significant new functionality. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+### Level 2: Product Analyst I
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.* | Influences and executes defined tasks and features across the data analytics stack. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives general instruction on day-to-day work, and detailed instruction on new assignments or unfamiliar tasks.<br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Receives general instruction on day-to-day work, and detailed instruction on new assignments or unfamiliar tasks.<br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
```

File: operations/research-and-development/product/analytics/analyst-levels.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+These levels serve to help Product Analyst assess their opportunities for career growth in the Product Analytics function. The levels below follow a growth track in the art and craft of Product Analysts and not in a person management track.
+
+The level descriptions below are transferrable to comparable analyst roles, such as a Product Analytics Engineer and a Data Analyst.
+
+### Level 1: Product Analyst I
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to show ownership.* | Executes defined tasks across the data analytics stack. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br> <br>Community: "How do others work with me?"<br> <br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives detailed instruction on most tasks. Asks appropriate questions, and knows when to stop and ask for help.<br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
```
Comment (justinegeffen) [inline operations/research-and-development/product/analytics/analyst-levels.md]:
```suggestion
| High Impact | Supervision: "How much do I rely on my manager?"<br> <br>Community: "How do others work with me?"<br> <br>*The need to receive supervision is inversely related to the instinct to choose what is high impact.* | Receives detailed instruction on most tasks. Asks appropriate questions, and knows when to stop and ask for help.<br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
```

**Review by justinegeffen — COMMENTED** [review]
Some minor editorial changes.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**